rst_files/markov_asset.rst

```diff
@@ -847,42 +765,38 @@ into vector :math:`Tw` via
     T w
     = \max \{ \beta M w,\; p - p_S {\mathbb 1} \}
 
-
 Start at some initial :math:`w` and iterate to convergence with :math:`T`
 
 We can find the solution with the following function `call_option`
 
-
 .. code-block:: julia
 
     # price of perpetual call on consol bond
-    function call_option(ap::AssetPriceModel, ζ::AbstractFloat, p_s::AbstractFloat, ϵ=1e-7)
+    function call_option(ap, ζ, p_s, ϵ = 1e-7)
 
-        # == Simplify names, set up matrices  == #
+        # Simplify names, set up matrices
```

Comment:
Comments should be lowercase

Reply:
Best to do that in another pass. I think most if not all comments at least in this lesson are camel case.